tests/tamr_client/utils.py

```diff
@@ -26,3 +26,29 @@ def dataset():
     url = tc.URL(path="datasets/1")
     dataset = tc.Dataset(url, name="dataset.csv", key_attribute_names=("primary_key",))
     return dataset
+
+
+def capture_payload(request, snoop, status, response_json):
+    """Capture request body within `snoop` so we can inspect that the request body is constructed correctly (e.g. for streaming requests).
+
+    See https://github.com/getsentry/responses#dynamic-responses
+    """
+    snoop["payload"] = list(request.body)
+    return status, {}, json.dumps(response_json)
+
+
+def records_to_deletes(records):
```

Comment:
@skalish what do you think about including the logic to create the DELETE command from a record within `tc.record`:

```python
>>> tc.record._delete_command({"primary_key": 1}, primary_key_name="primary_key")
{"action": "DELETE", "recordId": 1}
```
...and similarly for CREATE command?

Might be overkill, but it seems like a pretty record-specific functionality that doesn't belong in `tests.utils`

Reply:
I think that makes sense.  I'll give it a shot.

Reply:
@pcattori  Does it make sense to incorporate these into `tc.record.delete()` and `tc.record.create()`?

If so, I think this would deprecate `tc._delete_by_id()` as its only use (unless a user wants to access it directly) is to format the DELETE command.